### PR TITLE
chore(rpc): updates for PyPI saw-client release

### PR DESCRIPTION
**Pull request by pnwamk**

### CODE REVIEW

**Comment by RyanGlScott** [comment]
I think [this line](https://github.com/GaloisInc/saw-script/blob/fb24fed7a2d804e22d8fbe37d0713f0dbd951432/saw-remote-api/python/pyproject.toml#L2) in `pyproject.toml` needs to be updated as well.

**Review by atomb — APPROVED** [review]
Looks good!

File: saw-remote-api/python/pyproject.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
-name = "saw"
+name = "saw-client"
 version = "0.0.1"
```
Comment (RyanGlScott) [inline saw-remote-api/python/pyproject.toml:3]:
```suggestion
version = "0.8.0"
```

Perhaps?

Comment (pnwamk) [reply saw-remote-api/python/pyproject.toml]: Thanks!

**Review by RyanGlScott — APPROVED** [review]

**Pull request merged**